website/docs/plugins/submitit_launcher.md

```diff
@@ -23,73 +23,20 @@ Once installed, add `hydra/launcher=submitit` to your command line. Alternativel
 
 ```yaml
 defaults:
-  - hydra/launcher: submitit
+  - hydra/launcher: submitit_slurm
 ```
 
 Note that this plugin expects a valid environment in the target host. usually this means a shared file system between
 the launching host and the target host.
 
-Submitit supports 3 types of queues: auto, local and slurm. Its config looks like this
+Submitit actually implements 2 different launchers: `submitit_slurm` to run on a SLURM cluster, and `local` to test locally.
```

Comment:
What do you think about the following?
1. Include links to config.py with dataclasses instead of copying them in.
2. include the outputs of
```
$ python foo.py hydra/launcher=submitit_slurm --cfg hydra -p hydra.launcher
$ python foo.py hydra/launcher=submitit_local-cfg hydra -p hydra.launcher
```
When describing each of the two launchers.
I am still trying to discover what is the best way to document plugins with structured configs, and I think I like the above proposal.

Reply:
Ah, I saw that you already added similar output.
Also since the primary use case is SLURM we can focus on it.

This will output the current slurm parameters:
```
python foo.py hydra/launcher=submitit_slurm --cfg hydra -p hydra.launcher
```

and I recommend that instead of inlining the dataclasses we can just give a link.

Also, an example of how to override the launcher parameters from the command line would be helpful:

```
python foo.py hydra/launcher=submitit_slurm hydra.launcher.params.partition=prod
```

Reply:
> Ah, I saw that you already added similar output.

Yes I find them the most interesting since they make it easier to discover settings than just stating the config classes.

> and I recommend that instead of inlining the dataclasses we can just give a link.

I have shared feelings about this, since the interest of adding the config files there came from the description of each parameters. Then again, since it often ends up being outdated if not careful, it's probably better to just have the link.

So I just:
- removed the config classes
- added a link the the config classes
- added the discovery of submitit_local parameters
- added a commandline override example

Reply:
At some point I will support field description in the help or --cfg. when that happens we will be able a better description here.